tests/test_mongodb_full_table.py

```diff
@@ -212,17 +212,24 @@ def test_run(self):
         #  -------------------------------------------
         #  ----------- Second full Table Sync ---------
         #  -------------------------------------------
-        # add 2 rows and run full table again, make sure we get initial number + 2
-
         with get_test_connection() as client:
+        # update existing documents in the collection to make sure we get the updates as well in the next sync
```

Comment:
these comments are not indented to the right level

Reply:
done